main/appserver/tests/rq.sh

```diff
@@ -173,9 +173,10 @@ find_test_job(){
     return 1
 }
 
-USAGE="Usage: rq -b BRANCH -a ---> For runing all the test ids\n\
-	   or  rq -b BRANCH -g TESTGROUPNAME ---> For runing a TESTGROUPNAME\n\
-	   or  rq -b BRANCH -t TESTIDS ---> For runing a space separated set of TESTIDS"
+USAGE="Usage:\n 1) rq -l ---> List all the available TEST IDS without running them\n
```

Comment:
I suggest following (fixed some typos and re-formatted) 

Usage: 
 1) rq.sh -l ---> List all available test identifiers without running them
 2) rq.sh -b <branch> -a  ---> For running all tests
 3) rq.sh -b <branch> -g <test_group_name> ---> For running a test group
 4) rq.sh -b <branch> -t <test_id1 test_id2 test_id3> ---> For running a space separated list of tests